tox.ini

```diff
@@ -24,6 +27,8 @@ deps =
     pytest73: pytest==7.3.*
     pytest74: pytest==7.4.*
     pytestdev: git+https://github.com/pytest-dev/pytest#egg=pytest
+    py312: git+https://github.com/pytest-dev/pytest#egg=pytest
```

Comment:
This is redundant. You already use pytestdev with py312.

```suggestion
```

Reply:
oops copy/paste issue :sweat_smile: